build.sbt

```diff
@@ -11,11 +11,27 @@ val sbt10ScalaVersion     = "2.12.4"
 val libScalaVersion       = "2.11.12"
 val libCrossScalaVersions = Seq("2.11.8", "2.11.11", libScalaVersion)
 
-lazy val baseSettings = Seq(
-  organization := "org.scala-native",
-  version := nativeVersion
+// Convert "SomeName" to "some-name".
+def convertCamelKebab(name: String) : String = {
+  name.replaceAll( "([a-z])([A-Z]+)", "$1-$2" ).toLowerCase
+}
+
+// Generate project name from project id.
+def projectName(project: sbt.ResolvedProject) : String = {
+  convertCamelKebab(project.id)
+}
+
+// Provide consistent project name pattern.
+lazy val nameSettings = Seq(
+  normalizedName := projectName(thisProject.value), // Maven <artifactId>
+  name := s"Scala.native ${projectName(thisProject.value)}" // Maven <name>
```

Comment:
`Scala Native` is the official spelling, no dots.

Reply:
changed